### PR TITLE
improved documentation for connecting to web UIs

### PR DESCRIPTION
**Pull request by boegel**
rendered version @ http://boegel-hod.readthedocs.org/en/web_ui_docs/Connecting_to_web_UIs.html

### CODE REVIEW

**Comment by ehiggs** [comment]
What is this waiting on?

File: docs/Connecting_to_web_UIs.rst [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+Determine hostname of head node of HOD cluster
+**********************************************
+
+The first step is to figure out which workernode is the *head* node of your HOD cluster, using ``hod list``.
+
+For example::
+
+    $ hod list
+    Found 1 job Id 123456.master15.delcatty.gent.vsc State R Node node2001.delcatty.gent.vsc
```
Comment (boegel) [inline docs/Connecting_to_web_UIs.rst]: @ehiggs: please update this bit in your upcoming PR for `hod list`

File: docs/Connecting_to_web_UIs.rst [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+Setting up an SSH tunnel in Windows using PuTTy
+***********************************************
+
+http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html
+
+
+.. _browser_proxy_configuration:
+
+Browser SOCKS proxy configuration
+---------------------------------
```
Comment (boegel) [inline docs/Connecting_to_web_UIs.rst]: these bits are still empty...

**Comment by boegel** [comment]
Jenkins: test this please

File: docs/Connecting_to_web_UIs.rst (hunk 2 above)
Comment (boegel) [inline docs/Connecting_to_web_UIs.rst]: fixed now

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by ehiggs** [comment]
Jenkins: test this please.

**Comment by ehiggs** [comment]
Nice work!

**Pull request merged**